models/world_un_wpp/code/upload.py

```
@@ -0,0 +1,96 @@
+        path=str(path),
+        source_format="parquet",
+        if_table_exists="replace",
+        if_storage_data_exists="replace",
+    )
+    print("create+upload: OK", flush=True)
+
+    # Verify row count in BigQuery staging table
+    bq = bigquery.Client(project=BILLING_PROJECT)
+    query = (
+        f"select count(*) as n from "
+        f"`{BILLING_PROJECT}.{DATASET_ID}_staging.{table_id}`"
```

Ruff (0.15.20)

[error] Possible SQL injection vector through string-based query construction

(S608)